test/built-ins/Function/prototype/toString/class-declaration-explicit-ctor.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2016 Michael Ficarra. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-runtime-semantics-bindingclassdeclarationevaluation
+description: Function.prototype.toString on a class declaration (explicit constructor)
+---*/
+
+/* before */class /* a */ A /* b */ extends /* c */ B /* d */ { /* e */ constructor /* f */ ( /* g */ ) /* h */ { /* i */ ; /* j */ } /* k */ m /* l */ ( /* m */ ) /* n */ { /* o */ } /* p */ }/* after */
+
+assert.sameValue(A.toString(), "class /* a */ A /* b */ extends /* c */ B /* d */ { /* e */ constructor /* f */ ( /* g */ ) /* h */ { /* i */ ; /* j */ } /* k */ m /* l */ ( /* m */ ) /* n */ { /* o */ } /* p */ }");
+
+function B{}
```

Comment:
Thanks. This used to be a class declaration, but I wanted to avoid the TDZ error. I will add a parameter list.